services/director-v2/src/simcore_service_director_v2/cli/__init__.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import logging
+from typing import Final
+
+import typer
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from settings_library.utils_cli import create_settings_command
+
+from ..core.settings import AppSettings
+from ..meta import PROJECT_NAME
+from .core import async_node_save_state, async_project_save_state, async_project_state
+
+DEFAULT_NODE_SAVE_RETRY: Final[int] = 3
+DEFAULT_STATE_UPDATE_INTERVAL: Final[int] = 5
```

Comment:
If this is iin seconds, please set default to 10 or 15 seconds, I am afraid for our docker daemons :D

Reply:
Only the second one, it uses the same interval used by the fronted. I think it's ok, I query sequentially the status to avoid overloading the docker daemon.

Reply:
@GitHK add units as suffix or in comments so it is clear :-)